numpy/linalg/lapack_lite/make_lite.py

```diff
@@ -251,6 +261,7 @@ class F2CError(Exception):
     pass
 
 def runF2C(fortran_filename, output_dir):
+    fortran_filename = fortran_filename.replace('\\', '/')
```

Comment:
Why's this needed, and should it be using os.pathsep?

Reply:
f2c cannot understand `\\`, even if it's compiled on windows.

Reply:
You succeeded in running this on windows? Nice!